src/app/board/page.js

```diff
@@ -1,5 +1,27 @@
+import Board from "@/components/board/Board";
+import { BOARD } from "@/data/boardMembers";
+import Title from "@/components/Title";
+
 const Page = () => {
-  return <div>Board</div>;
+  return (
+    <div>
+      <div>
```

Comment:
remove the extra div tag

Reply:
![image](https://github.com/acm-ucr/h4b-website/assets/91030482/f06eb983-28a9-44bf-af5f-12bfb2644413)

try centering the elements in the parent tag using flexbox and items/justify alignment

Reply:
i would also try adding one more column